tsl/src/remote/dist_copy.c

```diff
@@ -813,9 +813,16 @@ remote_copy_process_and_send_data(RemoteCopyContext *context)
 	else
 		point = get_current_point_for_text_copy(context->data_context);
 
-	chunk = get_target_chunk(ht, point, &context->connection_state);
+	chunk = (Chunk *) get_target_chunk(ht, point, &context->connection_state);
```

Comment:
If we really need a mutable chunk here I think we should change `get_target_chunk` to return a mutable chunk  or move the mutating part into `get_target_chunk`.